### PR TITLE
K3s documention update

### PR DESCRIPTION
**Pull request by frozenprocess**
Added ubuntu 20.04 after testing
There was a point of confusion in documentation which hopefully I have clarified it now.

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 73fee45f42d59d9e8f21343fb86d973f5ea67736

https://deploy-preview-3712--calico.netlify.app

**Comment by frozenprocess** [comment]
I admit this commit got a bit out of hand !
It all started with Tigera operator, I thought it would be better if we could show both installations in a page. However, there was no tab pane integrated into documentations.

## how it works?
In your main markdown file add
```
{% include general-tab.html details='manifest:install:,operator:install:default' %}
```
> notice `:` structure that I had in mind was "tab name:tab content name:is it default or not" 

general-tab.html looks into `_includes/general-tab-details` folder and tries to find "tab name-tab content.md" file which will provide content for the tab pane.

**Comment by frozenprocess** [comment]
In commit [#08f9b5b](https://github.com/projectcalico/calico/pull/3712/commits/08f9b5b42757bad4cdc499011b96bf6ac52502ec) I have changed how users can implement tabs into their documentations.

in order to add a tab you can simply start a block `{% tabs %}`
each tab header can be populated with `<name=opreator>`
contents for tab should immediately follow header `<% content %>`
since this implementation leverages Jekyll block system you have to add `{% endtabs %}` when you are done.

there are some dynamic parameters that this block can accept. If you like to use pill buttons you can start the block and pass `pill` to it eg. `{% tabs pill %}`.
header accepts an `active` value which will active that item by default by setting appropriate `active` values into html .

File: getting-started/kubernetes/k3s/quickstart.md [hunk 1]
```diff
@@ -39,11 +39,44 @@ development.
...
+   K3s creates configuration files with `root` uid/gid, run the following command and change your user to `root`
 
+   ```
+   sudo su
+   ```
+
+### Install {{site.prodname}}
+
+   > **Note**: As of v3.15 milestone, manifest installation will gradually be replaced by {% include open-new-window.html text='Tigera operator' url='https://github.com/tigera/operator' %}.
+   > Tigera operator is another opensource solution from Tigera team which will deliver a better experience when you install or upgrade your
```
Comment (bmckercher123) [inline getting-started/kubernetes/k3s/quickstart.md:51]: open source

File: getting-started/kubernetes/k3s/quickstart.md [hunk 2]
```diff
@@ -81,6 +114,11 @@ development.
    ```
    {: .no-select-button}
 
+   %>
+   {% endtabs %}
+
+### Final Checks
```
Comment (bmckercher123) [inline getting-started/kubernetes/k3s/quickstart.md:120]: Final checks

File: getting-started/kubernetes/k3s/quickstart.md (hunk 1 above)
Comment (bmckercher123) [inline getting-started/kubernetes/k3s/quickstart.md:51]: Tigera operator is another open source solution from Tigera that delivers a better experience when you install or upgrade your

File: getting-started/kubernetes/k3s/multi-node-install.md [hunk 3]
```diff
@@ -63,9 +63,36 @@ We assume you have at least two instances of Linux (meeting the requirements) wi
 
 #### {{site.prodname}} Installation procedure
 
-   Install {{site.prodname}} by using the following command.
+   > **Note**: As of v3.15 milestone, manifest installation will gradually be replaced by {% include open-new-window.html text='Tigera operator' url='https://github.com/tigera/operator' %}.
```
Comment (bmckercher123) [inline getting-started/kubernetes/k3s/multi-node-install.md:66]: As v3.15,

File: getting-started/kubernetes/k3s/multi-node-install.md [hunk 4]
```diff
@@ -103,6 +130,11 @@ We assume you have at least two instances of Linux (meeting the requirements) wi
...
    {: .no-select-button}
 
+   %>
+   {% endtabs %}
+
+
+   
 #### Check the installation
    
 1. Confirm that all of the pods are running with the following command.
```
Comment (bmckercher123) [inline getting-started/kubernetes/k3s/multi-node-install.md:140]: Confirm that all of the pods are running using the following command.

File: getting-started/kubernetes/k3s/quickstart.md [hunk 5]
```diff
@@ -39,11 +39,44 @@ development.
    > CIDR by replacing 192.168.0.0/16 in the above command. 
    {: .alert .alert-danger}
 
+   K3s creates configuration files with `root` uid/gid, run the following command and change your user to `root`
```
Comment (bmckercher123) [inline getting-started/kubernetes/k3s/quickstart.md:42]: This sentence doesn't make sense. Two sentences?

 K3s creates configuration files with `root` uid/gid, run the following command and change your user to `root`

**Comment by bmckercher123** [comment]
Multi-node doc:
- Should follow How to template
- Typo in Note: "Remeber"
- Instead of "Calico installation procedure" should be "Install Calico"
- Mismatch between docs: one says "single-host cluster" other says "single-node cluster"

**Comment by frozenprocess** [comment]
> * Inconsistent use of "Notes" (one is green, one is blue, one is red).
Several color where used since each alert has different meaning.
Blue : ones where information that user can ignore.
Red : user needs to pay attention since it can raise an error.
Green : news

File: getting-started/kubernetes/k3s/quickstart.md (hunk 5 above)
Comment (frozenprocess) [reply getting-started/kubernetes/k3s/quickstart.md:42]: Thanks for pointing it out. 👍

**Comment by frozenprocess** [comment]
@bmckercher123 after carefully reading the page, I admit using green Note was a bit excessive :) thanks for point it out!

**Pull request merged**